opengate/managers.py

```diff
@@ -577,6 +589,51 @@ def dump_production_cuts(self):
             s += "*** No cuts per region defined. ***\n"
         return s
 
+    def add_surface(self, volume_1, volume_2, surface_name):
+        """
+        Adds a surface between volume_1 and volume_2 with specified finish.
+
+        Parameters:
+        volume_1 : The volume from.
+        volume_2 : The volume to.
+        surface_name : The name of the finish to be applied to the surface.
+
+        Returns:
+        None
+        """
+
+        surface_info = {"volumes": [volume_1, volume_2], "surface_name": surface_name}
+
+        volume_list = self.simulation.volume_manager.volumes
+
+        # Checks if the surface info of this volume pair already exists
+        # If exists, append. Else, create a new key pair
+        if volume_1 in volume_list and volume_2 in volume_list:
+            if volume_1 in self.volume_surfaces_info:
+                self.volume_surfaces_info[volume_1].append(surface_info)
+            else:
+                self.volume_surfaces_info[volume_1] = [surface_info]
+        else:
+            # If either volume_1 or volume_2 is not present, raise a fatal error
+            missing_volumes = [
+                vol for vol in [volume_1, volume_2] if vol not in volume_list
+            ]
+            fatal(
+                f"Volume(s) {', '.join(missing_volumes)} not present in created volumes"
+            )
+
+    def dump_surface_information(self):
```

Comment:
If we implement a class 
```python
class OpticalSurface(GateObject): 
...
```
with volume_1, volume_2, surface_name as user_info, we get a dump_info for free from the base class. 

(See general comment)